lib/charms/postgresql_k8s/v0/postgresql_tls.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
This library has been imported and can also be ignored, right?

Reply:
Yes, good catch! I forgot to add it to the pr description. Almost everything in the PR consists of imported files.